docs/user_guides/server-guide.md

```diff
@@ -84,6 +84,42 @@ Sample response:
 }]
 ```
 
+### Threads
```

Comment:
Should we also add a link to the sample config at the end of the section?

Reply:
Added.